cloudfour/cloudfour.com-patterns
# Fix spacing utility demos

## Overview

- Reuses postcss plugin config for both Webpack and Gulp tasks (not necessary for this update, but these were out of sync)
- Removes [postcss-preset-env](https://github.com/csstools/postcss-preset-env) plugin
- Updates spacing util demos to replace shorthand examples with `-start` and `-end` combinations, to fix issues across supported browsers. 
    - `postcss-preset-env` with [postcss-logical](https://github.com/csstools/postcss-logical) was unable to solve these issues and introduced others

## Screenshots

Utilities -> Spacing -> Padding demo, PR 651:

![image](https://user-images.githubusercontent.com/771447/80528193-65febd00-894a-11ea-8d32-864024f28170.png)

Same demo, this PR (note `.u-pad-block-1` and `.u-pad-inline-1`):

![image](https://user-images.githubusercontent.com/771447/80528265-829af500-894a-11ea-8ce3-6801b506e2aa.png)

## Testing

1. [Preview theme utils-spacing-padding preview](https://deploy-preview-662--cloudfour-patterns.netlify.app/?path=/story/utilities-spacing--padding)
1. [PR 651 utils-spacing-padding preview](https://deploy-preview-651--cloudfour-patterns.netlify.app/?path=/story/utilities-spacing--padding) - From before the introduction of `postcss-preset-env`
1. [Preview theme utils-spacing-margin preview](https://deploy-preview-662--cloudfour-patterns.netlify.app/?path=/story/utilities-spacing--margin)
1. [PR 651 utils-spacing-margin preview](https://deploy-preview-651--cloudfour-patterns.netlify.app/?path=/story/utilities-spacing--margin)
1. See that shorthand examples (`.u-pad-inline-1`, `.u-pad-block-1`, `.u-space-pad-1`, and `.u-space-block-1`) all look correct in the previews for this PR
1. Confirm in Chrome, FF, and Safari / iOS Safari (+ new Edge, if you like)

---

- Fixes #661

/CC @person
